Toolkit/ArcGISToolkit/JobManager.swift

```diff
@@ -222,26 +217,27 @@ public class JobManager: NSObject {
      `func application(_ application: UIApplication, performFetchWithCompletionHandler completionHandler: @escaping`
      */
     public func application(application: UIApplication, performFetchWithCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
-        if self.jobs.count > 0{
-            self.checkStatusForAllJobs{ completedWithoutErrors in
-                if completedWithoutErrors{
-                    completionHandler(.newData)
-                }
-                else{
-                    completionHandler(.failed)
-                }
-            }
+        guard jobs.count > 0 else {
```

Comment:
```suggestion
        guard !jobs.isEmpty else {
```

Although I believe performing work in the `else` clause goes against the spirit of `guard` because forward progress is made either way:

```swift
if !jobs.isEmpty {
    checkStatusForAllJobs { completedWithoutErrors in
        if completedWithoutErrors {
            completionHandler(.newData)
        } else {
            completionHandler(.failed)
        }
    }
} else {
    completionHandler(.noData)
}
```